config.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+// readConfigFile reads configuration files from /perm /etc or / and returns
+// trimmed content as string.
+//
+// TODO: de-duplicate this with gokrazy.go into a gokrazy/internal package
```

Comment:
Can you update the comment: doesn’t need to be gokrazy/internal, could even be useful for public packages